doc/design/vic-machine.md

```diff
@@ -63,6 +63,111 @@ To create a VCH, disambiguation of which resource to use from the available set
+
+### Fully specified command line
+
+```
+vic-machine create
+-TO=https://root@****:vcenter/
+-name=edw-web
+-compute-resource=/example-datacenter/host/mycluster/Resources/edw-pool/web
+-image-store=ds://datastore/containers/edw
+-container-store=ds://datastore/edw/edw/web
+-client-network="VM Network"
+-management-network="management-net"
+-volume-store=ds://edw-san/reports:reports
+-volume-store=ds://edw-san/data:data
+-docker-network=private-vlan12d7f2:bridge
+-docker-network=edw-corp-net:backend
+-docker-network-ipam=edw-corp-net:10.118.78.128-192/24,10.118.78.1
+-container=oracledb.edw.corp.net:db.backend
+```
+
+The first block of these options control the core configuration of the VCH:
+* name - the textual name by which this VCH is identified.
+* compute-resource - resource pool, host, cluster, or other compute boundry into which the VCH should be deployed.
+* image-store - the datastore location where images will be placed, whether pulled via `docker pull` or created via `docker commit`.
+* contianer-store - the datastore location where containerVMs will be created. This does not have to be on the same datastore as the images, but both must be visible to all hosts on which containerVMs are to be created.
+* client-network - the network on which users will connect to the VCH to issue DOCKER_API commands.
+* management-network - the network providing access to a VMOMI SDK, in deployments where access to the management network is required.
+
+The second block of options control how exisiting vSphere resources are presented to users of the VCH. These are specified as a `source:destination` mapping with vSphere identifier as the source; it is the viadmin role that is providing this data and mapping mydomain:targetdomain aligns with how people tend to think.
+* -volume-store - datastore prefixes under which volumes can be created. These prefixes are mapped to labels that can be reference via the --opts mechanism when calling `docker volume create`
+* -docker-network - vSphere networks that should be exposed via `docker network` commands. As with the datastores this is a mapping from vSphere name to docker name. Ideally the docker name should express something about the purpose of presenting the network such as `internet`, `intranet`, or `databases`
+* -docker-network-ipam - this is an optional argument furnishing additional information for controlling IP address management on the network, in the form `ipaddress-range/mask,gateway`. If not specified, DHCP is used.
+* -container - this allows containers to address a specific FQDN as if it were itself a container managed by the VCH, specified in the form of `FQDN:alias.network`.  
+
+
+### Manifest file - create
+Note that the various resource paths end in a `/` - this signifies that it's acceptable for additional refinement to be provided when using this manifest file:
+
+```
+vic-machine create
+-TO=file::///home/joe/vch/edw.manifest
```

Comment:
I've been unsure about this myself. I've been trying to figure out a way to maintain the symmetry of to/from so that it's possible to, for example, extract the config of an existing VCH and save it to a manifest, or the opposite.

I'm not happy with the current overloading, but am also reluctant to add yet another argument or command as we will rapidly lose the ease of use we're after. The best thought I've had so far is a simplification of your suggestion here, simply adding `manifest` as a new command to differentiation deploying of VCH from creating of manifest:
`vic-machine manifest ...`